pykubectl/objects.py

```diff
@@ -143,3 +176,30 @@ def execute(self, attempts=30):
     def logs(self, *args, **kwargs):
         cmd = f"logs {self.name}"
         return self.kubectl.execute(cmd, *args, **kwargs)
+
+class Job(Pod):
+    kind = "Job"
+
+    def execute(self, attempts=30):
+        logging.info("%s: execution initiated", self)
+
+        self.apply()
+
+        while attempts >= 0:
+            jobStatus = self.get()["status"]
+            if (isinstance(jobStatus.get("failed"), int) and jobStatus.get("failed") > 0):
+                colorRed = '\033[91m'
```

Comment:
colorization is nice but this is dependent on shell/TERM config no?

Reply:
ANSI color codes are pretty widely supported in terminals, and many support disabling them, in which case it just strips the color out. Since there are so many terminal options for Linux it's possible that maybe some older or perhaps intentionally minimal terminals may not have support, but I couldn't find a definitive list anywhere. 

I'm cool with stripping out the coloring though, I'll push up a change shortly

Reply:
Agreed, I didn't think it would break anything completely--might just give some weird output artifacts or something--but given that we don't have them currently, might be better to leave for later IMO.